src/server/getters/get-account-transfer-history.ts

```diff
@@ -1,7 +1,7 @@
 import * as Knex from "knex";
 import { Address, Bytes32 } from "../../types";
 
-interface TransferRow {
+export interface TransferRow {
```

Comment:
These seem internally used only, is there a reason to `export` them?

Reply:
Correct, `TransferRow` is internal and I won't export it. `TransferHistory` and `TransferLog` are in the public interface.